src/main/scala/com/microsoft/hyperspace/index/serde/package.scala

```diff
@@ -165,4 +166,30 @@ package object serde {
       override val isStreaming: Boolean)
       extends LeafNode
       with LogicalPlanWrapperWithInMemoryStates
+
+  case object CSVFileFormatWrapper extends FileFormat {
```

Comment:
Can we create `FileFormatWrapper` that implements these two functions?

Reply:
thanks @imback82 , added super class